tests/test_graph.py

```diff
@@ -176,6 +188,7 @@ def setUp(self):
         g.set_outputs((o1, o2))
         g.add_edges([(i1,v),(i2,w),(v,w),(v,o1),(w,o2)])
         self.i1, self.i2, self.v, self.w, self.o1, self.o2 = i1, i2, v, w, o1, o2
+        print(self.graph.scalar.to_number())
```

Comment:
Was this a debugging statement which was left in accidentally?

It causes the output of the test to look like this:
```
..................s............(1+0j)
.(1+0j)
.(1+0j)
.(1+0j)
.........................sssss.............................
----------------------------------------------------------------------
Ran 93 tests in 3.109s

OK (skipped=6)
```

Reply:
Sorry, yes I left it accidentally. I saw that you already removed it, thanks!